Source/AliveLib/Math.cpp

```diff
@@ -207,91 +207,51 @@ int CC Math_Distance_496EB0(int x1, int y1, int x2, int y2)
 }
 
 
-EXPORT FP CC Math_496F70(FP /*value1*/, FP /*value2*/)
+EXPORT FP CC Math_496F70(FP value1, FP value2)
 {
-    NOT_IMPLEMENTED();
-
-    /*
-    const FP v2 = value2;
-    FP* v3 = nullptr;
-    FP v16 = {};
-    if (value2 >= FP_FromInteger(0))
+    FP value1abs = value1;
+    FP value2abs = value2;
+    int switchScenario = 0;
+    if (value1 < FP_FromInteger(0))
     {
-        v3 = &value2;
+        value1abs = -value1;
+        switchScenario += 4;
     }
-    else
+    if (value2 < FP_FromInteger(0))
     {
-        v16 = -value2;
-        v3 = &v16;
+        value2abs = -value2;
+        switchScenario += 2;
     }
-
-    const FP v4 = *v3;
-    FP* v5 = nullptr;
-    if (value1 >= FP_FromInteger(0))
+    switchScenario += value2abs < value1abs;
```

Comment:
this should be made explicit instead of boolean implicit casting, use value2abs < value1abs ? 1 : 0